anndata/tests/test_base.py

```diff
@@ -298,6 +298,7 @@ def test_slicing_strings():
         adata[["A", "B", "not_in_obs"], :]
 
 
+# TODO: Decide what this should do
```

Comment:
Should these matrices be moved from .uns["neighbors"] at read time, or at instantiation time?